wp_api/src/users.rs

```diff
@@ -131,6 +131,21 @@ impl Display for WpApiParamUsersHasPublishedPosts {
     }
 }
 
+#[derive(
+    Debug, Clone, PartialEq, Eq, PartialOrd, Ord, Hash, Serialize, Deserialize, uniffi::Enum,
+)]
+#[serde(rename_all = "snake_case")]
+pub enum UserAvatarSize {
+    #[serde(rename = "24")]
+    Size24,
+    #[serde(rename = "48")]
+    Size48,
+    #[serde(rename = "96")]
+    Size96,
```

Comment:
Are these the only out-of-the-box sizes?

Reply:
I _think_ so: https://github.com/WordPress/WordPress/blob/caaf759d21dabffa84ab1e47d1a40798f2693fa0/wp-includes/rest-api.php#L1279-L1299